ftml/Cargo.lock

```diff
@@ -338,7 +338,6 @@ dependencies = [
  "lazy_static",
  "log",
  "maplit",
- "parking_lot",
```

Comment:
Why is this file getting changed?

Reply:
I don't know, actually. I tried to match the lock on develop (or whatever it is with slightly updated dependencies) but it does seem pretty different.